fix: LEAP-1963: Ignore is_labeled for ground_truth tasks

triklozoid opened the pull request:
* updated label_stream and lock logic to support `on-boarding`  mode

triklozoid committed:
fix: LEAP-1963: Ignore is_labeled for ground_truth tasks

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14134377694)
> Successfully merged: Already up to date.

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14222563816)
> Successfully merged:  create mode 100644 web/libs/core/src/hooks/useAbortController.ts

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1963'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14222563816

jombooth commented on label_studio/projects/functions/next_task.py:
```diff
@@ -179,7 +179,8 @@ def get_not_solved_tasks_qs(
         else:
-            not_solved_tasks = not_solved_tasks.filter(is_labeled=False)
+            if not project.show_ground_truth_first:
+                not_solved_tasks = not_solved_tasks.filter(is_labeled=False)
```
nit: seems cleaner to use `elif not project.show_ground_truth_first:`

triklozoid replied:
Fixed.

triklozoid committed:
Review fix

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14245725983)
> Successfully merged:  create mode 100644 web/libs/ui/src/shadcn.ts

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1963'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14245725983

MihajloHoma commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14304732621)
> Successfully merged:  create mode 100644 label_studio/users/product_tours/configs/show_ask_ai.yml

MihajloHoma committed:
Merge branch 'develop' into 'fb-LEAP-1963'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14304732621

MihajloHoma commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14304825214)

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14304825214

triklozoid committed:
Add feature flag

MihajloHoma commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14326361917)
> Successfully merged:  create mode 100644 docs/themes/v2/source/images/storages/gcs-project.png

MihajloHoma committed:
Merge branch 'develop' into 'fb-LEAP-1963'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14326361917

MihajloHoma commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14326438677)

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14326438677

triklozoid committed:
Fix label stream condition

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14360792384)
> Successfully merged:  create mode 100644 web/libs/editor/tests/integration/e2e/control_tags/from-prediction.cy.ts

triklozoid committed:
Merge branch 'develop' into 'fb-LEAP-1963'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14360792384
